### ISSUE TITLE
Are there pre-trained model parameters?

### ISSUE THREAD

**Issue by StevenAZy** [Ligo-Biosciences/AlphaFold3#40]
First of all, thank you very much for your excellent work. I would like to ask if there are any pre-trained models so that I don't have to retrain and can directly fine-tune.

**Comment by ardagoreci** [comment]
We have very early drafts of them, we will release the full weights once training and benchmarking is complete.

**Comment by tsjain** [comment]
Is your team closer to releasing working weights for the model?

**Comment by erikedlund** [comment]
@tsjain You will find more active development over at the [Boltz repository. ](https://github.com/jwohlwend/boltz)

**Comment by ardagoreci** [comment]
@tsjain yes, @erikedlund is right. This project is inactive, I will update the readme to reflect that.
